exchanges/order/order_types.go

```diff
@@ -129,7 +130,10 @@ type SubmitResponse struct {
 	Trades      []TradeHistory
 	Fee         float64
 	FeeAsset    currency.Code
-	Cost        float64
+	// Cost is the total cost of the order if hitting the bids it will be the base amount else lifting the asks quote amount
+	Cost float64
+	// Purchased is the amount of purchased currency hitting the bids will quote amount else lifting asks base amount
+	Purchased float64
```

Comment:
```suggestion
	Purchased float64 	// Buy in Quote currency, Sell in Base
```

Reply:
I'm fine with whatever it is, so long as the comment is clear 👍 
Just to check: Does it actually *need* to be directional?
Could Purchased always be in Base and Cost always be in Quote, regardless of side?